app/views/layouts/_sidebar.html.erb

```diff
@@ -65,13 +65,12 @@
           </li>
         <% end %>
         <% if all_casa_admin_signed_in? %>
-          <div class="list-group-item">
+          <li class="list-group-item">
             <%= current_all_casa_admin.email %>
-          </div>
+          </li>
           <%= link_to "Edit Profile", edit_all_casa_admins_path, class: "list-group-item" %>
         <% end %>
         <%= session_link %>
-        <li></li>
```

Comment:
It looks like this empty list element was added [here](https://github.com/rubyforgood/casa/commit/32efa1d20cf39ec260b7c55d418739ef510b2b29) as a way to combat a bootstrap bug. However, it doesn't look like we actually need it. Also there's a random floating bullet since this is a list element.